shlr/sdb/meson.build

```diff
@@ -58,6 +58,26 @@ sdb_exe = executable('sdb', 'src/main.c',
   implicit_include_directories: false
 )
 
+sdb_symlink_script = '''#script
```

Comment:
this should probably also go in https://github.com/radare/sdb, right @radare ?

Reply:
I will move it to another place.